Base: Add more manpages (v2) (#7611)
SerenityOS/serenity: Base/usr/share/man/man1/allocate.md

## Name

> allocate - allocate a memory

Reviewer: ```suggestion
allocate - allocate memory
```

## Synopsis

```**sh
$ allocate [number [unit (B/KiB/MiB)]]
```

## Description

> `allocate` allocates a specific amount of virtual memory (specified in `number` and `unit`, by default 50 MiB), It also tests write on each allocated page, then sleep for 10 seconds. It is primarily used to test kernel's memory management capabilities.

Reviewer: ```suggestion
`allocate` allocates a specific amount of virtual memory (specified in `number` and `unit`, by default 50 MiB), It also writes to each allocated page and then sleep for 10 seconds. It is primarily used to test the kernel's memory management capabilities.
```
